packages/xrpl/test/setupClient.ts

```diff
@@ -1,64 +1,96 @@
-/* eslint-disable no-param-reassign -- Necessary for test setup */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types -- Necessary for test setup */
-import { Client, BroadcastClient } from 'xrpl-local'
+import { Client } from '../src/client'
+// eslint-disable-next-line import/no-deprecated -- Will remove in 3.0.0
+import BroadcastClient from '../src/client/BroadcastClient'
 
-import createMockRippled from './createMockRippled'
-import { getFreePort } from './testUtils'
+import createMockRippled, {
+  type MockedWebSocketServer,
+} from './createMockRippled'
+import { destroyServer, getFreePort } from './testUtils'
+
+export interface XrplTestContext {
+  // eslint-disable-next-line import/no-deprecated -- Will remove in 3.0.0
+  client: Client | BroadcastClient
+  _mockedServerPort?: number
+  mockRippled?: MockedWebSocketServer
```

Comment:
Is this only optional because there could be multiple mocks? Seems like it'd be easier just to have a separate mock type for the BroadcastClient tests, so that every single call doesn't need the `!` in e.g. `testContext.mockRippled!`

Reply:
I'll add it to the followup